Sources/Parma/Render/ParmaRenderable.swift

```diff
@@ -56,7 +56,7 @@ public protocol ParmaRenderable {
     
     /// Define the style of list item.
     /// - Parameter view: The view contains view(s) which belong(s) to this item.
-    func listItem(view: AnyView) -> AnyView
+    func listItem(attributes: ListAttributes, index: [Int], view: AnyView) -> AnyView
```

Comment:
The comment of the function needs to update to include the explanation of index and attributes.

Reply:
Oops, done as well ✅